provisioning/datasources/default.yaml

```diff
@@ -9,6 +9,25 @@ datasources:
     uid: gdev-loki
     access: proxy
     url: http://host.docker.internal:3100
+    jsonData:
+      derivedFields:
+        - datasourceUid: gdev-jaeger
+          matcherRegex: traceID
+          matcherType: label
+          name: traceID-jaeger
+          url: '$${__value.raw}'
+          urlDisplayLabel: jaeger
+        - datasourceUid: gdev-zipkin
+          matcherRegex: traceID
+          matcherType: label
+          name: traceID-zipkin
+          url: '$${__value.raw}'
+        - datasourceUid: gdev-tempo
+          matcherRegex: traceID
+          matcherType: label
+          name: traceID-tempo
+          url: '$${__value.raw}'
+          urlDisplayLabel: tempo
```

Comment:
Did know about this, cool! These all might be related to the nature of the fake data

I aways think of traceIds opening with tempo or in Traces Drilldown. This one opened as a random walk in explore
<img width="1512" height="773" alt="Image" src="https://github.com/user-attachments/assets/ad80b6dd-206e-498a-91b0-d3df114a1ed8" />

This one opened in Traces explore but had an error, maybe expected?
<img width="1246" height="905" alt="Image" src="https://github.com/user-attachments/assets/8fb72e86-b659-4278-9d7e-cf0b09645a17" />

Reply:
For the tempo one: https://raintank-corp.slack.com/archives/C06M1RVAA0Z/p1752685814474789.
And yeah, the links won't generate correctly if the associated datasources don't exist, and we're not spinning up a jaeger or zipkin, but core generates the href so it's not something we should need to write test coverage for in the plugin, but if we set up a zipkin/jaeger datasource with those UIDs it should work.

Creating multiple mostly so we can assert that the link names and other stuff gets pulled in correctly.
Also when running server:ci the tempo isn't spun up so e2e tests need to be agnostic to the datasource getting created.